update methods to get the equivcells

bingyuew opened the pull request:
update methods to get the equivcells and update IR tables with the updated libcell func_ids

bingyuew committed:
update methods to get the equivcells and update IR tables with the updated libcell func_ids

Signed-off-by: Bing-Yue Wu <bywu@asu.edu>

VidyaChhabria reviewed (commented):
Bing-Yue, Could you summarize the changes you have done:
1) You have updated the scripts to read the libcell_id file because of the OpenROAD bug
2) You have updated the IR tables to use the libcell_id file because of the OpenROAD bug
3) You have updated the check valid script to check sequential cells are not sized

Some other comments: since we do not allow sizing of the registers. Can we only put one option for the FF in the libcell_id file? So the other cells are not options?

bingyuew commented:
1. Yes, I updated the OpenROAD example and helper scripts in the src/example directory and the evaluation scripts in the src/evaluation directory due to the OpenROAD bug.
2. I also updated the IR tables to use the libcell_id file because of the OpenROAD bug.
3. is mentioned in (1)
4. I have addressed this in the check_valid script: if the inst in the pre-sized def file is a sequential cell, it cannot be swapped to use other sequential cells.
